_data/committers.yml

```diff
@@ -42,15 +42,15 @@
 - name: Antoine Pitrou
   role: PMC
   alias: apitrou
-  affiliation: Voltron Data
+  affiliation: "-"
```

Comment:
This page uses "Independent" in some places and, newly, "-" to mean what I think is the same thing. Would it make sense to use one or the other consistently?

Reply:
To me, "Independent" can mean something precise like independent contractor. I'll let other people correct their own entries, but my current affiliation is undetermined :-)

Reply:
WFM.